sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -449,14 +449,17 @@ with Text Analytics and require Python 3.5 or later.
 Authenticate the client with a Cognitive Services/Text Analytics API key or a token credential from [azure-identity][azure_identity]:
 * [sample_authentication.py][sample_authentication] ([async version][sample_authentication_async])
 
-In a batch of documents:
+Common scenarios with the different endpoints
```

Comment:
nit: can we use a different word than "endpoints" here? Maybe even just "Common scenarios"?